locales-pending/dashboard-premium.ftl

```diff
@@ -0,0 +1,173 @@
+  You’ll need to manually request removal from each site.
+modal-exposure-type-data-broker-part-two = If you’re a { -brand-premium } user, we automatically remove all profiles for you.
+  In both cases, removals typically take 7-14 days. Some can take longer, while others can happen within the hour.
+
+# About Exposure Statuses Modal
+
+# Variables:
+# $data_broker_sites_total_num is the total number of data broker sites available to scan. It will always be plural.
+modal-exposure-status-description-premium = We search for exposures in all known data breaches and { $data_broker_sites_total_num } data broker sites that sell your personal info.
+  Your exposures will have one of the following statuses:
+modal-exposure-status-in-progress = <b>In progress</b> means we are actively working on fixing the exposure for you. This is a { -brand-premium } feature.
+
+# Variables:
+# $exposures_unresolved_num is the unresolved number of exposures the user has.
+# $data_breach_unresolved_num is the unresolved number of data breaches the user has.
+# $data_broker_unresolved_num is the unresolved number of data brokers selling the user’s data.
+dashboard-exposures-area-description-premium = We found your information exposed { $exposures_unresolved_num } times over { $data_breach_unresolved_num } data breaches and { $data_broker_unresolved_num } data broker sites that are selling your personal info.
+
+# Variables:
+# $exposures_unresolved_num is the unresolved number of exposures the user has.
+# $data_breach_unresolved_num is the unresolved number of data breaches the user has.
+dashboard-exposures-breaches-scan-progress-description = {
+  $exposures_unresolved_num ->
+    [one] {
+      $data_breach_unresolved_num ->
+        [one] We found your information exposed { $exposures_unresolved_num } time in { $data_breach_unresolved_num } data breach. We’re still scanning sites that may be selling your personal info.
+        *[other] We found your information exposed { $exposures_unresolved_num } time in { $data_breach_unresolved_num } data breaches. We’re still scanning sites that may be selling your personal info.
+    }
+    *[other] {
+      $data_breach_unresolved_num ->
+        [one] We found your information exposed { $exposures_unresolved_num } times in { $data_breach_unresolved_num } data breach. We’re still scanning sites that may be selling your personal info.
+        *[other] We found your information exposed { $exposures_unresolved_num } times in { $data_breach_unresolved_num } data breaches. We’re still scanning sites that may be selling your personal info.
+    }
+}
+dashboard-exposures-no-breaches-scan-progress-description = We didn’t find any data breaches, but we’re still scanning sites that may be selling your personal info.
+dashboard-exposures-scan-progress-label = Scan in progress
+
+dashboard-fixed-area-headline-premium = View all exposures that are fixed or in-progress
```

Comment:
We don't seem very consistent in how to spell "in progress" vs "in-progress"?

Reply:
I believe (but I'm not a native speaker) that "in-progress" is an adjective (i.e. "an in-progress scan"), whereas "in progress" is a sentence fragment ("the scan is in progress").

Reply:
Right, but we're only using in-progress in two places (the category itself is "In progress", sometimes title case).

The other usage is even weirder

> This chart does not include any exposures that are in-progress of being auto-removed.

Reply:
Asked about that [here](https://mozilla.slack.com/archives/CEV7MGSQH/p1704359607505119).

Do we want to wait for an answer, or can we fix it up later? (Since I don't expect it to influence translations, right?)

Reply:
Anything in `locales-pending` can be fixed later.

Reply:
Ah heh right, I'll get to rebasing then :)